### ISSUE TITLE
Ability to change accessibility volume easily

### ISSUE THREAD

**Issue by PatrykMis** [Irineu333/SpeakTouch#94]
Draft suggestion is to do this by the following ways at once:

1. While SpeakTouch is speaking something, let the volume buttons always increase or decrease accessibility volume; and
2. while holding one or two (to be discussed) fingers and then pressing volume button, let accessibility volume change regardless whether SpeakTouch is speaking or not.

**Comment by Irineu333** [comment]
Interesting, actually changing the accessibility volume is a pain.

**Comment by PatrykMis** [comment]
Exactly. We can again check how it's done in TalkBack and of course simplify it (both are implemented), though in iOS only 1. is implemented.
